### ISSUE TITLE
dtrace-provider unable to be installed on iojs 2.0.1 and 2.1.0

### ISSUE THREAD

**Issue by toddself** [trentm/node-bunyan#258]
This isn't a bug with bunyan per se, but I'm getting a bunch of those superfluous errors when I start up my app:

```
{ [Error: Cannot find module './build/Release/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/default/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
{ [Error: Cannot find module './build/Debug/DTraceProviderBindings'] code: 'MODULE_NOT_FOUND' }
```

Filed on node-dtrace-provider as [issue 64](https://github.com/chrisa/node-dtrace-provider/issues/64).

**Comment by stephank** [comment]
This was just now fixed in dtrace-provider 0.5. (Safe upgrade, nothing else but a NAN upgrade.)

**Comment by trentm** [comment]
Thanks! Sorry for the delay. Should be in bunyan@1.4.0.
